### PR TITLE
dev-dtvf-cesium-styling

### PR DESCRIPTION
**Pull request by mdhillman**
This PR brings in a new version of the DTVF with the below changes:

- Cesium styling expressions can now be used in `data.json` file
  - Ths is detailed in the "Layers" section of the README in the example Cesium visualisation
  - Example Cesium visualisation has been updated with an example style expression
- Fixed issue with removing highlights on selected Cesium features

This PR has been marked as a draft whilst other DTVF PRs take precedence (#741 and #517)

### CODE REVIEW

**Review by sm453 — CHANGES_REQUESTED** [review]
Tested. Styling works fine.

One thing, though: The clipping planes side panel appears just fine in v3.4.0, but when upgrading to v3.5.0, everything appears bold and in the large heading font, and it is no longer functional. Is that simply because the changes from v3.4.0 have not been merged into this branch?

**Comment by jlai2021** [comment]
A newly reported issue, #791. This issue could potentially have an impact on our current testing. As such, I suggest we pause the testing process until this issue is properly addressed and resolved.

File: web/digital-twin-vis-framework/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+### Features:
+* Added ability to specify client-side styling & filtering options for Cesium 3D tilesets.
+* Updated sample NYC data used in example Cesium visualisation.
+### Bug Fixes:
+* Restored missing JS file used to build legends in example visualisations.
+* Minor CSS tweaks to UI components.
+
 # 3.4.0
 ### Features:
 * New clippling planes feature to allow `vertical` slicing of 3D tile sets.
```
[the remark this answers is not in the document]
Comment (mdhillman) [reply web/digital-twin-vis-framework/CHANGELOG.md:13]: Yes, will change.

**Comment by mdhillman** [comment]
CSS issues causing clipping plane controls to disappear have now been resolved. Could @jlai2021 please test this an confirm at earliest convenience?

File: web/digital-twin-vis-framework/example-cesium-vis/webspace/index.html [hunk 2]
```diff
@@ -161,8 +161,8 @@
 
             // ===== CUSTOMISABLE =====
             // Enter Mapbox account name and API key here, even in Cesium visualisations!
-            MapHandler.MAP_USER = "";
-            MapHandler.MAP_API = "";
+            MapHandler.MAP_USER = "cmclinnovations";
+            MapHandler.MAP_API = "pk.eyJ1IjoiY21jbGlubm92YXRpb25zIiwiYSI6ImNrbGdqa3RoNDFnanIyem1nZXR3YzVhcmwifQ.hVk983r6YYlmFE8kSMbzhA";
```
Comment (sm453) [inline web/digital-twin-vis-framework/example-cesium-vis/webspace/index.html:165]: Please remove user and API key.

**Review by sm453 — CHANGES_REQUESTED** [review]
Tested again. Now everything works as expected, including the clipping planes.

File: web/digital-twin-vis-framework/example-cesium-vis/webspace/index.html (hunk 2 above)
Comment (mdhillman) [reply web/digital-twin-vis-framework/example-cesium-vis/webspace/index.html]: Now resolved.

**Review by sm453 — APPROVED** [review]

**Pull request merged**